web/src/app/listing/component.html

```diff
@@ -16,19 +16,19 @@
 		    	<div class="course-details">{{creditRange}}</div>
 		    </tr>
 			</table>
-			<span *ngIf="showDescriptionTooltip" class="description-tooltip position-absolute" placement="right" [ngbPopover]="listing.description" container="body" triggers="mouseenter:mouseleave">
+			<span *ngIf="showDescriptionTooltip" class="description-tooltip position-absolute" placement="right" [ngbPopover]="tooltipDescription" container="body" triggers="mouseenter:mouseleave">
 				<img src="assets/images/info.svg" />
 			</span>
 			<span (click)="descriptionClick()" *ngIf="!showDescriptionTooltip" class="course-description" [class.hideDescription]="!showingDescription" [class.showDescription]="showingDescription" [class.selectedDescription]="isCourseSelected()">
 				{{listing.description}}
 			</span>
 			<div class="menu position-absolute" (click)="showingMenu=!showingMenu">
-				<div class = "arrow-container down-arrow" *ngIf="!showingMenu">
```

Comment:
what's the reason for this change?

Reply:
uhhh..... actually I'm not sure. I never changed any of these lines except 19 myself.

Reply:
Please revert this change and I'll merge this :)

Reply:
done